src/app/page.tsx

```diff
@@ -75,12 +104,37 @@ export default function Home() {
 									<Badge className="mt-1 mb-3 text-sm">{member.role}</Badge>
 								</div>
 								<p className="text-sm">
-									<strong>Fact 1:</strong> Here comes fact 1.
+									<strong>Coffee Order:</strong>{" "}
+									{member.coffeeOrder || "Not provided"}
 								</p>
 								<p className="text-sm">
-									<strong>Best Tech Tool:</strong> Here comes fact 2.
+									<strong>Favorite Tech-Tool:</strong>{" "}
+									{(
+										<a className="hover:underline" href={member.techToolLink}>
+											{member.techTool}
+										</a>
+									) || "Not provided"}
 								</p>
-								<p className="text-sm mt-5">My social media accounts.</p>
+								<div className="mt-4">
+									<ul className="flex flex-wrap items-center mb-6 text-sm font-medium text-gray-500 sm:mb-0 dark:text-gray-400">
+										<li>
+											<a
+												href={member.instaLink}
+												className="hover:underline me-4 md:me-6"
+											>
+												Insta
+											</a>
+										</li>
+										<li>
+											<a
+												href={member.linkedInLink}
+												className="hover:underline me-4 md:me-6"
+											>
+												LinkedIn
+											</a>
+										</li>
+									</ul>
+								</div>
```

Comment:
If the code for the card showing the person's is to big we could separate it into its own component.
That way when we add extra functionality like the "show young photo" then everything related to that component is encapsulated